Gemfile

```diff
@@ -71,20 +71,21 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem "byebug", platforms: [:mri, :mingw, :x64_mingw]
   # Adds support for Capybara system testing and selenium driver
-  gem "rspec-rails", "~> 4.0.2"
+  gem "rspec-rails", "~> 6.0.3"
```

Comment:
would be nice to document the reason for this somewhere, is this needed?

Reply:
this is more of a nice to have, it's not necessary, but i figured we might want to update this dependency while we are bumping dependencies. everything seems to work but it is NOT a required dependency bump for the rails upgrade.